hack/functest.sh

```diff
@@ -43,3 +43,30 @@ TEST_NAMESPACE=node-maintenance-operator go test ./test/e2e/... -root=$(pwd) -ku
 
 echo "E2e tests passed"
 
+echo "check validation of openaAPIV3Schema"
+
+$KUBECTL_CMD create -f _out/namespace-init.yaml
+
+$KUBECTL_CMD create -f _out/nodemaintenance_crd.yaml
+
+echo "validate CRD"
+
+VALIDATE_CRD=$($KUBECTL_CMD get -o yaml crd nodemaintenances.kubevirt.io)
```

Comment:
I think you need the `set +e` logic here too

Reply:
added || true in both cases (to set exit status of pipeline to success in any case)